dags/glam.py

```diff
@@ -209,18 +261,26 @@
         arguments=('--append_table','--noreplace',),
         dag=dag)
     clients_histogram_aggregates_merged_0 >> clients_histogram_aggregates_merged
-    clients_histogram_aggregates_merged >> clients_histogram_aggregates
+    clients_histogram_aggregates_merged >> clients_histogram_aggregates_0
 
-clients_histogram_bucket_counts = bigquery_etl_query(
-    task_id="clients_histogram_bucket_counts",
-    destination_table="clients_histogram_bucket_counts_v1",
-    dataset_id=dataset_id,
-    project_id="moz-fx-data-shared-prod",
-    owner="msamuel@mozilla.com",
-    email=["telemetry-alerts@mozilla.com", "msamuel@mozilla.com"],
-    date_partition_parameter=None,
-    arguments=('--replace',),
-    dag=dag)
+    clients_histogram_aggregates = bigquery_etl_query(
+        task_id="clients_histogram_aggregates_{}".format(partition),
+        destination_table="clients_histogram_aggregates_v1",
+        dataset_id=dataset_id,
+        project_id="moz-fx-data-shared-prod",
+        owner="msamuel@mozilla.com",
+        email=["telemetry-alerts@mozilla.com", "msamuel@mozilla.com"],
+        date_partition_parameter=None,
+        parameters=(
+            "min_sample_id:INT64:{}".format(min_param),
+            "max_sample_id:INT64:{}".format(max_param),
+        ),
+        arguments=('--append_table','--noreplace',),
+        dag=dag)
+    clients_histogram_aggregates_0 >> clients_histogram_aggregates
+    clients_histogram_aggregates >> clients_histogram_bucket_counts
+    clients_histogram_aggregates >> glam_valid_builds
```

Comment:
Valid builds are needed before the clients_daily queries.

Reply:
> Valid builds are needed before the clients_daily queries.

Good point, it's nicer to have them ready right before using them instead of preparing them in advance